qhub/stages/input_vars.py

```diff
@@ -60,6 +60,9 @@ def stage_02_infrastructure(stage_outputs, config):
                     "instance_type": value["instance"],
                     "min_size": value["min_nodes"],
                     "max_size": value["max_nodes"],
+                    "guest_accelerators": value["guest_accelerators"]
```

Comment:
Change to interface: 

```
aws_gpu_enabled: True/False
gcp_gpu_enabled: True/False
```

And calculate if enabled in python instead of terraform `any(node_group.gpu for node_group in node_groups)`.

Reply:
Hi @costrouc I updated the PR based on this, would you mind reviewing this?